controllers/ssp_controller.go

```diff
@@ -55,6 +57,14 @@ var sspOperands = []operands.Operand{
 	// TODO - add other operands here
 }
 
+// 1:1 list of legacy CRDs corresponding to the operands above
```

Comment:
If you extract the `PauseCRs` functions from the operands, this list will not need to be 1:1 mapping.